frontend/src/components/DiagnosticSummary/QualityDiagnosticValue.vue

```diff
@@ -20,6 +20,9 @@ export default {
     toCurrency(value) {
       return toCurrency(value)
     },
+    isTruthyOrZero(value) {
```

Comment:
Eventually factorisable to `utils` (along with `percentageDisplay`)

Reply:
https://github.com/betagouv/ma-cantine/issues/4039